server/master/src/main/java/org/apache/accumulo/master/tableOps/MoveExportedFiles.java

```diff
@@ -48,21 +55,35 @@
 
       Map<String,String> fileNameMappings = PopulateMetadataTable.readMappingFile(fs, tableInfo);
 
-      for (String oldFileName : fileNameMappings.keySet()) {
-        if (!fs.exists(new Path(tableInfo.exportDir, oldFileName))) {
-          throw new AcceptableThriftTableOperationException(tableInfo.tableId, tableInfo.tableName,
-              TableOperation.IMPORT, TableOperationExceptionType.OTHER,
-              "File referenced by exported table does not exists " + oldFileName);
-        }
-      }
+      FileStatus[] exportedFiles = fs.listStatus(new Path(tableInfo.exportDir));
+      FileStatus[] importedFiles = fs.listStatus(new Path(tableInfo.importDir));
+
+      Set<String> importing = Arrays.stream(exportedFiles).map(fstat -> fstat.getPath().getName())
+        .map(fileNameMappings::get).collect(Collectors.toSet());
 
-      FileStatus[] files = fs.listStatus(new Path(tableInfo.exportDir));
+      Set<String> imported = Arrays.stream(importedFiles).map(fstat -> fstat.getPath().getName())
+        .collect(Collectors.toSet());
```

Comment:
`fstat -> fstat.getPath().getName()` is used twice. The lines can be shortened (possibly to one line each) if that lambda was saved in a reusable `Function<FileStatus,String> fileStatusName = fstat -> fstat.getPath().getName();` variable or similar.

Reply:
Addressed in latest commit - however FYI the formatter still causes the lines to wrap.